tests/testthat/test-descriptives.R

```diff
@@ -1,8 +1,35 @@
 context("Descriptives")
 
+# https://jasp-stats.github.io/jasp-verification-project/descriptives.html
+test_that("Main table results match R, SAS and SPSS", {
+  options <- jaspTools::analysisOptions("Descriptives")
+  options$variables <- "Data"
+  options$mean <- TRUE
+  options$median <- TRUE
+  options$variance <- TRUE
+  options$mode <- FALSE
+  options$standardDeviation <- TRUE
+  options$standardErrorMean <- TRUE
+  options$minimum
+  options$maximum
+  options$mode <- TRUE
+  options$percentileValuesQuartiles <- TRUE
+  
+  # Main table
+  results <- jaspTools::runAnalysis("Descriptives",
+                                    data.frame("Data"=1:5),
+                                    options)
+  resultTable <- results$results$stats$data
+
+  jaspTools::expect_equal_tables(
+    "test"=resultTable,
+    "ref"=list(5, 3, 3, 1, 0, 1, 1.58113883008419, 0.707106781186548, 5, "Data",
+               2.5, 2, 3, 4)
+  )
+})
```

Comment:
So because we have modules, you could also make a different file and do something like:

```r
# test-descriptivesVerification.R (or some other name)
context("Descriptives -- Verification project")

# all tests for the verification project
```
because otherwise the test files might get very long

Reply:
Great idea!